PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py

```diff
@@ -310,13 +298,20 @@ def miniAOD_customizeCommon(process):
         process.heepIDVarValueMaps.elesMiniAOD = cms.InputTag('reducedEgamma','reducedGedGsfElectrons')
         #force HEEP to use miniAOD (otherwise it'll detect the AOD)
         process.heepIDVarValueMaps.dataFormat = cms.int32(2)
-        
 
+        #add the HEEP trk isol to the slimmed electron
+        process.slimmedElectrons.modifierConfig.modifications[0].electron_config.heepV70TrkPtIso = cms.InputTag("heepIDVarValueMaps","eleTrkPtIso")
```

Comment:
this simple index ```[0]``` access is not safe against changes in what gets loaded in egammaObjectModificationsInMiniAOD_cff.py.
Some logic should be added to select a correct "modifications" element (it looks like the target is EGExtraInfoModifierFromFloatValueMaps).

Also, can this go in the default place RecoEgamma/EgammaTools/python/egammaObjectModificationsInMiniAOD_cff.py ?

Reply:
so this is now fixed, I thought I had already, just missed it. 

I cant put it in RecoEgamma/EgammaTools/python/egammaObjectModificationsInMiniAOD_cff.py as its conditional on the variable existing in the first place (unlike MVA folks, I dont run which doesnt need to be run). Meh its not great but I dont want to waste time on coming up with a better solution right now.